README.adoc

```diff
@@ -36,33 +37,35 @@ Documentation follows a link:https://redhat-documentation.github.io/modular-docs
 │   │        ├── title-1.adoc
 │   │        ├── title-2.adoc
 │   │        └── modules -> ../modules/
-│   └── release-notes
+├── release-notes
 └── README.adoc
 ....
 
 Top-level directories are:
 
-* downstream: A directory that stores the Red Hat Ansible Automation platform modules, assemblies, and titles.
+* `common`: Contains modules and attributes that are shared across Red Hat Ansible Automation branches.
+You should modify content in the `common` folder on the `main` branch only.
 
-Lower-level directories are:
+* `downstream`: Contains Red Hat Ansible Automation platform modules, assemblies, and titles in these sub-folders:
 
-* attributes: A common directory for attribute files.
-* assemblies: A directory for all assemblies, each on in its own AsciiDoc file. Assemblies are collections of modules linked together.
-** Assemblies are grouped by platform component. Create a new subdirectory to organize assemblies associated with that component.
-* images: A directory for all images and other digital content.
-* modules: A directory for all modules, each one in its own AsciiDoc file.
-** Modules are grouped by platform component. Create a new subdirectory to organize modules associated with that component.
-* titles: A directory for titles (information units, e.g, install guide, user guide). Contains master.adoc files to be called during builds and symbolic links to directories (modules, etc.) that contain .adoc content.
-** Titles are grouped by platform component or platform-level feature. Create a new subdirectory to organize titles associated with a component.
+** `attributes`: A common directory for attribute files.
+** `assemblies`: A directory for all assemblies, each on in its own AsciiDoc file. Assemblies are collections of modules linked together.
+*** Assemblies are grouped by platform component. Create a new subdirectory to organize assemblies associated with that component.
+** `images`: A directory for all images and other digital content.
+** `modules`: A directory for all modules, each one in its own AsciiDoc file.
+*** Modules are grouped by platform component. Create a new subdirectory to organize modules associated with that component.
+** `titles`: A directory for titles (information units, e.g, install guide, user guide). Contains master.adoc files to be called during builds and symbolic links to directories (modules, etc.) that contain .adoc content.
+*** Titles are grouped by platform component or platform-level feature. Create a new subdirectory to organize titles associated with a component.
```

Comment:
Can we rename this TItles_Platform" or something that distinguishes this from the lower level title manual titles?

Reply:
Sure, do you want to push changes to my branch?

Reply:
That, and I forgot how to do it.